!Command
command:
- fargene_model_creation
positional: []
named: []
parent:
subcommands: []
usage: []
help_flag:
usage_flag:
version_flag:
help_text: "Traceback (most recent call last):\n  File \"/usr/local/bin/fargene_model_creation\"\
  , line 11, in <module>\n    load_entry_point('fargene==0.1', 'console_scripts',\
  \ 'fargene_model_creation')()\n  File \"/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 489, in load_entry_point\n    return get_distribution(dist).load_entry_point(group,\
  \ name)\n  File \"/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 2843, in load_entry_point\n    return ep.load()\n  File \"/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 2434, in load\n    return self.resolve()\n  File \"/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 2440, in resolve\n    module = __import__(self.module_name, fromlist=['__name__'],\
  \ level=0)\n  File \"/usr/local/lib/python2.7/site-packages/fargene_model_creation/create_and_optimize_model.py\"\
  , line 9, in <module>\n    from calculate_performance import calculate_performance,\
  \ summarize_sens_or_spec\n  File \"/usr/local/lib/python2.7/site-packages/fargene_model_creation/calculate_performance.py\"\
  , line 2, in <module>\n    from plot_cross_validation import plot_cross_validation\n\
  \  File \"/usr/local/lib/python2.7/site-packages/fargene_model_creation/plot_cross_validation.py\"\
  , line 2, in <module>\n    import matplotlib.pyplot as plt\n  File \"/usr/local/lib/python2.7/site-packages/matplotlib/pyplot.py\"\
  , line 115, in <module>\n    _backend_mod, new_figure_manager, draw_if_interactive,\
  \ _show = pylab_setup()\n  File \"/usr/local/lib/python2.7/site-packages/matplotlib/backends/__init__.py\"\
  , line 63, in pylab_setup\n    [backend_name], 0)\n  File \"/usr/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py\"\
  , line 4, in <module>\n    from . import tkagg  # Paint image to Tk photo blitter\
  \ extension.\n  File \"/usr/local/lib/python2.7/site-packages/matplotlib/backends/tkagg.py\"\
  , line 5, in <module>\n    from six.moves import tkinter as Tk\n  File \"/usr/local/lib/python2.7/site-packages/six.py\"\
  , line 203, in load_module\n    mod = mod._resolve()\n  File \"/usr/local/lib/python2.7/site-packages/six.py\"\
  , line 115, in _resolve\n    return _import_module(self.mod)\n  File \"/usr/local/lib/python2.7/site-packages/six.py\"\
  , line 82, in _import_module\n    __import__(name)\n  File \"/usr/local/lib/python2.7/lib-tk/Tkinter.py\"\
  , line 39, in <module>\n    import _tkinter # If this fails your Python may not\
  \ be configured for Tk\nImportError: libX11.so.6: cannot open shared object file:\
  \ No such file or directory\n"
generated_using:
- --help
docker_image:
